src/browser.ts

```diff
@@ -6,6 +7,8 @@ import {
   type RenderResult
 } from './mermaid-isomorphic.js'
 
+mermaid.registerLayoutLoaders(mermaidElk)
+
```

Comment:
I prefer not to do this in the browser implementation, as it modifies the global Mermaid state. Instead, we should document in the readme that ELK is only supported in Node.js, and the user can add this if they want browser support.

```suggestion
mermaid.registerLayoutLoaders(mermaidElk)

```

Reply:
done in 9556ea57dec471b9ded2487213ff1eb14668224d and 4649e91a43fdf83e22a0730e5c906bff26cb6df9